features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeatureTest.kt

```diff
@@ -184,13 +183,10 @@ internal class SessionReplayFeatureTest {
 
         // Then
         verify(lambda).invoke(captor.capture(), any(), any())
-        if (ResourcesFeature.RESOURCES_ENDPOINT_ENABLED) {
-            assertThat(captor.firstValue)
-                .isInstanceOf(SessionReplayResourcesWriter::class.java)
-        } else {
-            assertThat(captor.firstValue)
-                .isInstanceOf(NoOpResourcesWriter::class.java)
-        }
+        // always a noop, since even when the resource feature flag is enabled
+        // it only changes to the implementation datawriter after onInitialise
```

Comment:
but there is `onInitialize` call above, which contradicts this comment

Reply:
This was a bit of an issue for me in the test, because the relevant `onInitialise` is part of the `ResourcesFeature`, and here we're testing the `SessionReplayFeature`. The RF is also created inside SRF, so I don't hold a reference to call `onInitialise` on